app/routes/intro.js

```diff
@@ -1,4 +1,8 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
+  model: function(){
+     var submission = this.store.createRecord('submission', {data_url: ''});
```

Comment:
@meghanhade  I found out that default value becomes `api/responses` instead of blank sentence if I create the record without this parameter.

Reply:
@hanbyul-here that's ok, this part will change again as soon as the changes for hitting the web service url here are merged in.